frontend/src/pages/team/Devices/dialogs/ProvisioningCredentialsDialog.vue

```diff
@@ -46,6 +52,19 @@ export default {
         close () {
             this.$refs.dialog.close()
             this.token = undefined
+        },
```

Comment:
This is used in multiple places and I can see us using it in more, should it be a mixin?

Reply:
Indeed it should but then that would be touch code in other pages too so out of scope. But to answer the question definitely yes (via separate issue perhaps)

Reply:
Actually, scratch that, you are right.  Since I'm adding it twice, that should be enough to warrant common code. Will look at making this a mixin tomorrow.

Reply:
resolved in commit [4b00087](https://github.com/flowforge/flowforge/pull/2231/commits/4b0008762fa7960e821c37fcaf0f8f177883e774)